.github/workflows/docusaurus-staging.yml

```
@@ -35,13 +35,21 @@ jobs:
           DEST_DIR: "$DOCUSAURUS_BASE_URL"
       - run: |
           echo "Hosted at: ${DOCUSAURUS_URL}/${DOCUSAURUS_BASE_URL}/index.html"
-      - name: Comment on PR
-        uses: actions/github-script@0.3.0
+      - name: Find Comment
+        uses: peter-evans/find-comment@v1
+        id: fc
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const { issue: { number: issue_number }, repo: { owner, repo }  } = context;
-            const {DOCUSAURUS_URL, DOCUSAURUS_BASE_URL} = process.env;
-            const body = `Staging build complete! Visit it at: ${DOCUSAURUS_URL}/${DOCUSAURUS_BASE_URL}/index.html`;
-            github.issues.createComment({ issue_number, owner, repo, body });
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: SDK Docs Site Staging Build
+      - name: Output Results to PR Comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            # SDK Docs Site Staging Build
```

Hmm the h1 is a bit big with the underline. Btw we could fancy it up with some emoji too :)
```suggestion
            ## SDK Docs Site Staging Build
```